### PR TITLE
consensus service & admin service use new DI pattern

### PR DESCRIPTION
**Pull request by hendrikebbers**
consensus service refactored for new DI pattern
admin service refactored for new DI pattern
minimal changes in token service to match same pattern

### CODE REVIEW

File: hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/handlers/AdminHandlers.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * A collection of all admin handlers
+ */
+@Singleton
+public class AdminHandlers {
```
Comment (netopyr) [inline hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/handlers/AdminHandlers.java:28]: I think this class makes only sense, if we plan to add more handlers to this module. @kimbor, what do you think?

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java
Comment (netopyr) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java:1]: The plan was to get rid of `TransactionHandlers` and `QueryHandlers`, right? Maybe we can already start to remove handlers from these classes, once they are available through service-specific `XyzHandlers` classes.

Comment (hendrikebbers) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java:1]: I would like to do that once all services are refactored to not make the PRs too big...

File: hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/handlers/AdminHandlers.java (hunk 1 above)
Comment (hendrikebbers) [reply hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/handlers/AdminHandlers.java:28]: Let's do the same pattern for all services first of all

**Review by iwsimon — APPROVED** [review]
LGTM for the consensus service changes.

**Review by Neeharika-Sompalli — APPROVED** [review]

File: hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/AdminServiceModule.java [hunk 2]
```diff
@@ -14,19 +14,19 @@
...
-    interface Factory {
-        AdminComponent create();
-    }
+/**
+ * Dagger module of the admin service
+ */
+@Module
+public interface AdminServiceModule {
 
     FreezeHandler freezeHandler();
```
Comment (rbair23) [inline hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/AdminServiceModule.java:29]: Should this method be removed now?

**Review by rbair23 — APPROVED** [review]

File: hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/AdminServiceModule.java (hunk 2 above)
Comment (hendrikebbers) [reply hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/AdminServiceModule.java:29]: That is something we should discuss once all services are refactored: Do we want to allow to inject individual handlers or only the `FooHandlers` component to access all handlers of a service?

**Pull request merged**